src/coord/src/coord.rs

```diff
@@ -774,6 +776,14 @@ impl Coordinator {
                             },
                         }
 
+                        if let Err(e) = check_statement_safety(&stmt) {
```

Comment:
Shouldn't this check be conditional on safe mode being enabled?

Reply:
D'oh, yes.